### DISCUSSION TITLE
Feature Brainstorm: What New Capabilities Would Enhance Our Project?

### DISCUSSION THREAD

**Discussion by bishoy-at-pieces** [pieces-app/cli-agent#194] [Ideas]
# 🚀 Feature Suggestion Thread

Hello!

We're always looking for ways to improve our project and make it more useful for everyone. That's where you come in! We'd love to hear your ideas for new features or enhancements that you think would make a significant impact.

## 💡 Share Your Ideas

Please use this thread to suggest new features you'd like to see implemented. When proposing a feature, consider including:

1. A clear, concise description of the feature
2. Examples of how it might be used (if applicable)

**Comment by stuartsc** [comment]
Amazing experience so far. Although context isn't 100%, and I'm having issues logging into Outlook because I signed up using google with the same work address, it still seems to capture enough info to be very useful.

Would love the capability to rewind screen and audio much like rewind.ai

**Reply by bishoy-at-pieces** [reply]
Hey @stuartsc 
Thank you for your feedback!
Can I know more what is wrong with the context?

Also can you please report any login issue or any other suggest to our support repo
https://github/pieces-app/support
